### PR TITLE
feat: rewards usd values

### LINKED ISSUE #140: In the Rewards section/token dropdown list  (token icon is not responsive)
https://github.com/user-attachments/assets/d1d98174-91d1-48ea-a2f4-76a335eace56

Screenshot 2024-08-27 at 16 03 40

### PR DESCRIPTION
**Pull request by guerrap**
Also fixes #140.

### CODE REVIEW

File: packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx [hunk 1]
```diff
@@ -66,6 +67,13 @@ export function RewardsStep({
         rewardToken?.address,
     );
 
+    const totalRewardsUsdAmount = useMemo(() => {
+        if (!rewards) return 0;
+        return rewards.reduce((accumulator, reward) => {
+            return (accumulator += reward.amount * reward.price);
```
Comment (luzzif) [inline packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx:73]: What happens here if `price` is null?

File: packages/frontend/src/components/create-campaign/preview/index.tsx [hunk 2]
```diff
@@ -52,6 +52,11 @@ export function CampaignPreview({
     const { address: connectedAddress } = useAccount();
     const { writeContractAsync } = useWriteContract();
 
+    const campaignDurationDays = useMemo(() => {
+        if (!payload.startDate) return 0;
```
Comment (luzzif) [inline packages/frontend/src/components/create-campaign/preview/index.tsx:56]: One thing here would be that no one guarantees us that the campaign will last more than a day. So for that look how I did it in the campaign row in the all campaigns page. That's more flexible.

**Review by luzzif — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx (hunk 1 above)
Comment (guerrap) [reply packages/frontend/src/components/create-campaign/form/rewards-step/index.tsx]: Fixed in a06f031f998ba0c7d995b892b4e399b596601e9d.

File: packages/frontend/src/components/create-campaign/preview/index.tsx (hunk 2 above)
Comment (guerrap) [reply packages/frontend/src/components/create-campaign/preview/index.tsx]: Fixed in a06f031f998ba0c7d995b892b4e399b596601e9d.

**Review by luzzif — APPROVED** [review]

**Pull request merged**